aqua/FScript/Portfile

```diff
@@ -28,15 +28,12 @@ master_sites            https://github.com/pmougin/F-Script/zipball/v${version}:
 distfiles               pmougin-F-Script-v2.1-0-g25c850c.zip:fscript
 
 checksums               pmougin-F-Script-v2.1-0-g25c850c.zip \
```

Comment:
This may be a candidate for a [stealth update](https://trac.macports.org/wiki/PortfileRecipes#stealth-updates)?